tools/dts.js

```diff
@@ -205,44 +209,48 @@ function parseExportFrom(content, currentFileName, queue, baseDir, projDir) {
     return content.replace(regExportFrom, '');
 }
 
+function parseImportFrom(content, currentFileName, queue, baseDir, projDir) {
+    var matches;
+    while ((matches = regImportFrom.exec(content))) {
+        var fromFileName = parseFrom(matches[1].trim(), currentFileName, baseDir, projDir);
+        enqueue(queue, fromFileName);
+    }
+    return content.replace(regImportFrom, '');
+}
+
 function process(baseDir, queue, index, projDir) {
     var item = queue[index];
     var currentFileName = item.filename;
     var file = fs.readFileSync(currentFileName);
     var content = file.toString();
 
-    // 1. Remove single line comments
-    content = content.replace(singleLineComment, '');
-
-    // 2. Process 'export ... from ...;'
+    // 1. Process 'export ... from ...;'
     content = parseExportFrom(content, currentFileName, queue, baseDir, projDir);
 
-    // 3. Remove imports
-    content = content.replace(regImportFrom, '');
-
-    // 4. Parse classes and interfaces
-    content = parseClasses(content, item.elements);
-
-    // 5. Parse functions
-    content = parseFunctions(content, item.elements);
-
-    // 6. Parse enum
-    content = parseEnum(content, item.elements);
-
-    // 7. Parse type
-    content = parseType(content, item.elements);
+    // 2. Remove imports
```

Comment:
is there no need to parse type, functions, classes and interfaces?

Reply:
I think he is parsing everything that has the `export` statement

Reply:
Yes, that is on line 234